conf/messages.cy

```diff
@@ -154,7 +154,7 @@ giveInformation.hint.true = Give information.
 giveInformation.hint.false = Give information (optional).
 giveInformation.error.required = Enter information
 giveInformation.error.length = Information must be 350 characters or less
-giveInformation.error.character = Information must contain letters or numbers
+giveInformation.error.character = Information does not contain any letters or numbers
```

Comment:
Content updated in V2 of AR-04 and is the same as this ticket too so I reused it